CONTRIBUTING.md

```diff
@@ -134,26 +141,52 @@ For additional information on using the `dep` tool, see
 
 ## Patch format
 
+### General format
+
 Beside the `Signed-off-by` footer, we expect each patch to comply with the
 following format:
 
 ```
-Subsystem: One line change summary
+subsystem: One line change summary
+
+More detailed explanation of your changes (why and how)
+that can span as many lines as required.
+
+A "Fixes #XXX" comment listing the github issue this change resolves.
+This is required for the main patch in a sequence - see the examples below.
```

Comment:
Suggested rewrite: This comment is required for the main patch in a sequence. See the following examples.